Update user deactivation documentation for v10.10 offline status changes (#8122)
mattermost/docs: source/onboard/user-provisioning-workflows.rst

Once deactivated, users still exist in the Mattermost database and their messages can still be viewed in Mattermost. You can use the :ref:`mmctl <manage/mmctl-command-line-tool:mmctl user delete>` tools to delete a user and all of their content.

> From Mattermost v10.10, when a user is deactivated, their their :ref:`availability <preferences/set-your-status-availability:set your availability>` is automatically set to offline.

Author: ```suggestion
From Mattermost v10.10, when a user is deactivated, the account's :ref:`availability <preferences/set-your-status-availability:set your availability>` is automatically set to offline.
```

> Starting from Mattermost server v10.10, when a user is deactivated, their availability status is automatically set to offline to ensure accurate status reporting and improve the user experience.

Author: ```suggestion
From Mattermost v10.10, when a user is deactivated, their their :ref:`availability <preferences/set-your-status-availability:set your availability>` is automatically set to offline.
```
